backend/internal/db/db.go

```diff
@@ -26,5 +26,5 @@ func ConnectSupabaseDB() (*pgx.Conn, error) {
 	}
 
 	fmt.Println("Successfully connected to the database!")
-	return conn, nil
+	return conn, err
```

Comment:
why change this? if it has gotten this far, there is no err. better to explicitly state that
```suggestion
	return conn, nil
```